src/vs/workbench/common/theme.ts

```diff
@@ -118,6 +118,12 @@ export const STATUS_BAR_FOREGROUND = registerColor('statusBar.foreground', {
 	hc: '#FFFFFF'
 }, nls.localize('statusBarForeground', "Status bar foreground color. The status bar is shown in the bottom of the window."));
 
+export const STATUS_BAR_NO_FOLDER_FOREGROUND = registerColor('statusBar.noFolderForeground', {
+	dark: '#FFFFFF',
```

Comment:
I suggest to use `STATUS_BAR_FOREGROUND` here as default color for all themes instead of specifying the color again as `FFFFFF`. This makes sure that all existing themes are not broken that specify `STATUS_BAR_FOREGROUND` any different from `FFFFFF`.